src/einsteinpy/symbolic/einstein.py

```diff
@@ -10,7 +11,9 @@ class EinsteinTensor(BaseRelativityTensor):
     Class for defining Einstein Tensor
     """
 
-    def __init__(self, arr, syms, config="ll", parent_metric=None):
+    def __init__(
+        self, arr, syms, config="ll", parent_metric=None, name="EinsteinTensor"
```

Comment:
```suggestion
        self, arr, syms, config="ll", parent_metric=None, name=None
```
Let this remain `None` for now. Why discrimination between `ChristoffelSymbols` and `EinsteinTensor` ? 
😃

Reply:
User can inspect the class to get the Tensor sub-type. No need for default `EinsteinTensor` in my opinion

Reply:
Okay, I would take back my suggestion. Let it remain as it is